lib/patch-series.ts

```diff
@@ -288,6 +303,12 @@ export class PatchSeries {
                                 }
                             });
                             break;
+                        case "range-diff:":
+                            if (doRangeDiff) {
+                                throw new Error(`Duplicate version-diff`);
+                            }
+                            doRangeDiff = match2[2].toLowerCase() === "yes" ? true : false;
```

Comment:
Maybe we should use `true` here?

A later PR might imitate Git's config setting strategy where some settings can be Boolean (and which accept `true` and `false` but not `yes`), but also contain more information. For example, I imagine that we could allow passing arbitrary `range-diff` options, such as `--creation-factor=9999` or `-U30`.

Reply:
Okay.  Converting to a string.  This affects the next comment.